src/context.rs

```diff
@@ -212,120 +212,120 @@ mod tests {
     #[test]
     fn new_with_too_little_mem_cost_returns_correct_error() {
         let mem_cost = 7;
-        let result = Context::new(Variant::Argon2i,
-                                  Version::Version13,
-                                  mem_cost,
-                                  3,
-                                  4,
-                                  ThreadMode::Sequential,
-                                  vec![0u8; 8],
-                                  vec![0u8; 8],
-                                  Vec::with_capacity(0),
-                                  Vec::with_capacity(0),
-                                  32);
-        assert_eq!(result, Err(Error::MemoryTooLittle));
+        assert_eq!(Context::new(Variant::Argon2i,
```

Comment:
The extra `result` variable is removed because the generated `Context` with reference can not live longer than the temporary `Vec`s.